dist/src/main/java/io/camunda/zeebe/broker/StandaloneBroker.java

```diff
@@ -33,7 +33,11 @@
  */
 @SpringBootApplication(
     proxyBeanMethods = false,
-    scanBasePackages = {"io.camunda.zeebe.broker", "io.camunda.zeebe.shared"})
+    scanBasePackages = {
+      "io.camunda.zeebe.broker",
+      "io.camunda.zeebe.shared",
+      "io.camunda.zeebe.gateway.rest"
```

Comment:
:thought_balloon: Not sure about the package, because it means the gateway will always automatically load it. That's probably fine though.

Reply:
The current requirement is that we can toggle the gRPC server, but the REST API server will always be enabled since some of the APIs will only be available there and not in gRPC.

It might be a problem if we ever decide to add the possibility to disable the REST server, but I don't see this as happening.

Reply:
Any thoughts? :thinking:

Reply:
I think that's fine. We added the `@ConditionalOnRestGatewayEnabled` annotation which should ensure that no REST API controllers are loaded if the embedded gateway is disabled.